src/style.css

```diff
@@ -2,0 +1,54 @@
+body {
+  margin: 0;
+}
+
+.container {
+  box-sizing: border-box;
+  background-color: rgba(255, 255, 255, 1);
+  display: flex;
+  flex-direction: column;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 div:nth-child(1) {
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). In this case, you have `margin-right` on `.stars__star`. If you need to add space between elements inside `.stars`, consider using `margin-left` or `margin-right` consistently on all child elements.